pkg/apiserver/rest/usecase/rbac.go

```diff
@@ -0,0 +1,783 @@
+/*
+Copyright 2022 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package usecase
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+	"sync"
+
+	"github.com/emicklei/go-restful/v3"
+
+	"github.com/oam-dev/kubevela/pkg/apiserver/datastore"
+	"github.com/oam-dev/kubevela/pkg/apiserver/log"
+	"github.com/oam-dev/kubevela/pkg/apiserver/model"
+	apisv1 "github.com/oam-dev/kubevela/pkg/apiserver/rest/apis/v1"
+	"github.com/oam-dev/kubevela/pkg/apiserver/rest/utils/bcode"
+	"github.com/oam-dev/kubevela/pkg/utils"
+)
+
+// resourceActions all register resources and actions
+var resourceActions map[string][]string
+var lock sync.Mutex
+var reg = regexp.MustCompile(`(?U)\{.*\}`)
+
+var defaultProjectPermPolicyTemplate = []*model.PermPolicyTemplate{
+	{
+		Name:      "app-management",
+		Alias:     "App Management",
+		Resources: []string{"project:${projectName}/application:*/*"},
+		Actions:   []string{"*"},
+		Effect:    "Allow",
+		Level:     "project",
```

Comment:
Maybe `Scope` will be better of `Level`?

Reply:
agree